process/throttle/antiflood/quotaFloodPreventer_test.go

```diff
@@ -164,7 +247,68 @@ func TestNewQuotaFloodPreventer_IncrementUnderMaxValuesShouldIncrementAndReturnT
 	assert.True(t, putWasCalled)
 }
 
-func TestNewQuotaFloodPreventer_IncrementOverMaxNumMessagesShouldNotPutAndReturnFalse(t *testing.T) {
+func TestNewQuotaFloodPreventer_IncrementAddingSumWithResetShouldWork(t *testing.T) {
+	t.Parallel()
+
+	putWasCalled := 0
```

Comment:
this sounds like a boolean variable. Maybe rename to `numPutOperations`

Reply:
changed